src/java/picard/analysis/CollectGcBiasMetrics.java

```diff
@@ -59,12 +59,31 @@
  * edited by Kylee Bergin
  */
 @CommandLineProgramProperties(
-        usage = "Tool to collect information about GC bias in the reads in a given BAM file. Computes" +
-                " the number of windows (of size specified by WINDOW_SIZE) in the genome at each GC%" +
-                " and counts the number of read starts in each GC bin.  What is output and plotted is" +
-                " the \"normalized coverage\" in each bin - i.e. the number of reads per window normalized" +
-                " to the average number of reads per window across the whole genome..\n",
-        usageShort = "Collects information about GC bias in the reads in the provided SAM or BAM",
+        usage = "Tool to collect information about GC bias in the reads in a given BAM file." +
+                "<br /> <br /> " +
+                "Guanine and cytosine (GC) base pairs have three hydrogen bonds compared with two for" +
```

Comment:
Right now it's not obvious why this information is included, beyond the fact that it has to do with the difference between GC and AT pairs. Can you clarify how this relates to what the tool does and how to interpret results?